app/views/comments/_comment_thread.html.erb

```diff
@@ -9,7 +9,7 @@
       <% if child_comments && child_comments.count > 0 %>
         <!-- cut off the recursion when we get too deep, but not if there's just one last measly comment left -->
         <% if depth >= ArchiveConfig.COMMENT_THREAD_MAX_DEPTH && child_comments.count > 1 %>
-          <li class="comment"><p>(<%= ts("%{count} more comments in this thread", :count => comment.children_count) %>)</p></li>
+          <li class="comment"><p>(<%= link_to ts("%{count} more comments in this thread", :count => comment.children_count), comment_url(comment) %>)</p></li>
```

Comment:
Could you update this to use `count: comment.children_count` rather than the hash rocket?